sklearn_numba_dpex/kmeans/engine.py

```diff
@@ -91,14 +97,14 @@ def __init__(self, estimator):
         self.order = order
         self.estimator = estimator
 
-        _testing_mode = os.getenv("SKLEARN_NUMBA_DPEX_TESTING_MODE", "0")
-        if _testing_mode not in {"0", "1"}:
+        _is_testing_mode = os.getenv("SKLEARN_NUMBA_DPEX_is_testing_mode", "0")
+        if _is_testing_mode not in {"0", "1"}:
             raise ValueError(
-                "If the environment variable SKLEARN_NUMBA_DPEX_TESTING_MODE is set, "
-                f'it is expected to take values in {"0", "1"}, but got {_testing_mode}'
-                " instead."
+                "If the environment variable SKLEARN_NUMBA_DPEX_is_testing_mode is"
+                f' set, it is expected to take values in {"0", "1"}, but got'
+                f" {_is_testing_mode} instead."
             )
-        self._testing_mode = _testing_mode == "1"
+        self._is_testing_mode = _is_testing_mode == "1"
```

Comment:
Ah, I meant the following which I think reads better.
```suggestion
        self._is_in_testing_mode = _is_testing_mode == "1"
```

WDYT?